tests/calc/files/parameters/test_checker.py

```diff
@@ -7,6 +7,8 @@
 __created__ = '2023-10-25'
 __license__ = 'GPLv3+'
 
+from unittest.mock import MagicMock
```

Comment:
It doesn't really matter as they are compatible, but is there a way to do this in pytest?

Reply:
Thanks for the input. Indeed, there is [pytest-mock](https://pytest-mock.readthedocs.io/en/stable/index.html), which makes mocking way nicer. I've added it as an additional test dependency and updated the tests. 1bc4e3932c70da5e076ff045ee6122224bd0a855